### ISSUE TITLE
How to reproduce the accuracy of RTMDet-tiny

### ISSUE THREAD

**Issue by Robert-JunWang** [open-mmlab/mmyolo#667]
### Prerequisite

- [X] I have searched [the existing and past issues](https://github.com/open-mmlab/mmyolo/issues) but cannot get the expected help.
- [X] I have read the [FAQ documentation](https://mmyolo.readthedocs.io/en/latest/faq.html) but cannot get the expected help.
- [X] The bug has not been fixed in the [latest version](https://github.com/open-mmlab/mmyolo).

### 💬 Describe the reimplementation questions

It is fantastic that RTMDet-tiny achieves 41.0 mAP on the MS COCO dataset. However, When I train it with the default config file on the MS COCO dataset. I only get 37.4 mAP.  Are there any parameters that I should change to reproduce the result? Thanks a lot. The model is trained on a single RTX 3090 GPU (24GB).

### Environment

PyTorch: 1.13.1
GPU: Single RTX 3090

**Comment by hhaAndroid** [comment]
@Robert-JunWang If your total batch size is not the same as the one we provide, you need to go to the adaptive scaling learning rate.

**Comment by Robert-JunWang** [comment]
Thanks a lot for your reply. If I train on a single GPU with batch size 32, Should I change the base_lr from 0.004 to 0.0005? Are there any other parameters that need to be changed? I am confused about the default value of the base_lr in rtmdet_l_syncbn_fast_8xb32-300e_coco.py. Is it for 8xb32 or 8xb16?

```
# Base learning rate for optim_wrapper. Corresponding to 8xb16=64 bs
base_lr = 0.004
```

**Comment by hhaAndroid** [comment]
@Robert-JunWang the default value of the base_lr in rtmdet_l_syncbn_fast_8xb32-300e_coco.py. It is for 8xb32
